main.go

```diff
@@ -349,7 +357,7 @@ func prepareFlags(c *cli.Context) error {
 	}
 	format.InitializeFormatter(c.String("formatter"), os.Stdout)
 
-	if config.IsHost || config.BrownfieldToken != "" {
+	if config.IsHost || config.BrownfieldToken != "" || (config.CommandPollingToken != "" && config.ServerID != "") {
```

Comment:
I would remove the requirement of the config.ServerID being set in order to consider the CLI is acting as a server agent, with the CommandPollingToken is enough. I would still have it validate on the register command that the server id is set as you have done in https://github.com/ingrammicro/concerto/pull/16/files#diff-6cb8eab5075806bffe5e076fd996a6fbR94